# Servo parameters
# https://github.com/mike4192/spotMicro/blob/master/servo_calibration.md#description-of-servo-configuration-values
num_servos: 12
servo_max_angle_deg: 82.5
# NOTE 1: 'center_angle_deg' MUST NOT BE GREATHER THAN 'servo_max_angle_deg'
# NOTE 2: 'range' must be less or equal than 'center'
#
# center: 306 (12-bit PWM) = 1.5 uS pulse width
# Equation:
#  B = D * f * 4096 / 1E6
# Where:
# - B: 12-bit digital value
# - D: duty cycle
# - f: PWM frequency (50 Hz)
#
# RF_3: {num: 1,  center: 306, range: 312, direction: -1, center_angle_deg:  -90.0}
# RF_2: {num: 2,  center: 306, range: 330, direction: -1, center_angle_deg:   -8.5}
# RF_1: {num: 3,  center: 306, range: 440, direction: -1, center_angle_deg:   11.3}
# RB_3: {num: 4,  center: 306, range: 295, direction: -1, center_angle_deg: -102.9}
# RB_2: {num: 5,  center: 306, range: 350, direction: -1, center_angle_deg:   -4.7}
# RB_1: {num: 6,  center: 306, range: 216, direction: -1, center_angle_deg:   -0.8}
# LB_3: {num: 7,  center: 306, range: 330, direction: -1, center_angle_deg:   60.0}
# LB_2: {num: 8,  center: 306, range: 337, direction: -1, center_angle_deg:    0.0}
# LB_1: {num: 9,  center: 306, range: 297, direction: -1, center_angle_deg:  -16.7}
# LF_3: {num: 10, center: 306, range: 348, direction: -1, center_angle_deg:   32.2}
# LF_2: {num: 11, center: 306, range: 312, direction: -1, center_angle_deg:  -10.6}
# LF_1: {num: 12, center: 306, range: 367, direction: -1, center_angle_deg:  -13.5}

RF_3: {num: 1,  center: 306,range: 378,direction:  1, center_angle_deg: 75.15}
RF_2: {num: 2,  center: 306,range: 372,direction:  1, center_angle_deg: -1.77}
RF_1: {num: 3,  center: 306,range: 345,direction: -1, center_angle_deg: 16.76}
RB_3: {num: 4,  center: 306,range: 381,direction:  1, center_angle_deg: 72.26}
RB_2: {num: 5,  center: 306,range: 369,direction:  1, center_angle_deg: -8.27}
RB_1: {num: 6,  center: 306,range: 359,direction:  1, center_angle_deg: -4.4}
LB_3: {num: 7,  center: 306,range: 369,direction:  1, center_angle_deg: -89.55}
LB_2: {num: 8,  center: 306,range: 336,direction:  1, center_angle_deg: -23.11}
LB_1: {num: 9,  center: 306,range: 345,direction: -1, center_angle_deg: 20.11}
LF_3: {num: 10, center: 306,range: 369,direction:  1, center_angle_deg: -93.58}
LF_2: {num: 11, center: 306,range: 361,direction:  1, center_angle_deg: -12.34}
LF_1: {num: 12, center: 306,range: 367,direction:  1, center_angle_deg: -2.70}
